packages/SwingSet/src/kernel/state/storageWrapper.js

```diff
@@ -76,17 +86,34 @@ export function buildCrankBuffer(kvStore) {
       assert.typeof(value, 'string');
       additions.set(key, value);
       deletions.delete(key);
+      if (isConsensusKey(key)) {
+        crankhasher.add('add');
+        crankhasher.add('\n');
+        crankhasher.add(key);
+        crankhasher.add('\n');
+        crankhasher.add(value);
+        crankhasher.add('\n');
+      }
```

Comment:
What's the motivation for laying it out this way?  I think I would have written
```
crankhasher.add(`add ${key} ${value}\n`);
```
Why use newline as the internal separator, why use the same separator internally and between entries, and why add the separators with separate `add` calls?  Not saying the way you did it is wrong, just curious.